src/Symfony/Component/HttpKernel/Tests/Fixtures/KernelForTest.php

```diff
@@ -34,4 +34,9 @@ public function isBooted()
     {
         return $this->booted;
     }
+
+    public function registeredDependencies()
+    {
+        return parent::registeredDependencies();
```

Comment:
Yep, because the parent one is `protected` so redefining it here as `public` allows to get dependencies outside the class.